client/wfnews-war/src/main/angular/src/app/components/report-of-fire/location-page/rof-location-page.component.html

```diff
@@ -1,6 +1,14 @@
 <div class="page-content">
   <div class="rof-text-primary-header">{{ title }}</div>
   <div class="rof-text-secondary-header">{{ message }}</div>
+  <div>
```

Comment:
Not sure if we need to bring in SMK for the location map as it doesn't have any actual map functionality? Might make more sense to just use a basic leaflet implementation (see the incident details header map)

Reply:
Im not sure if leaflet can drag the map with arrow and my location ?  Also the appearance of scale tool in leaflet are diferent as SMK. 
I got the refrence from PM and thought there was a reason to use SMK.

Reply:
I'm not saying "dont use SMK", just wondering if it's too heavy-weight for the page. If we want specific functionality that it provides though, no big deal to utilize it.